sap/5.3/modules/ROOT/pages/sap-connector-examples.adoc

```diff
@@ -169,70 +344,64 @@ Example response:
  			<CRETIM>050305</CRETIM>
  			<SERIAL>20191004050305</SERIAL>
  		</EDI_DC40>
-
  		...
-
-    ```
 ----
 
-=== XML to Send an IDoc to SAP
+== Receive an SAP Function
 
-[source,xml,linenums]
+This example shows you how to create a Mule app that receives an SAP function. The following screenshot shows the Studio flow for this example:
+
+image::sap-connector-example-establish-connection.png[Studio flow for establishing an SAP connection]
+
+To create the flow:
+
+. From the Mule Palette view, select *SAP* and drag the *Function listener* source to the canvas.
+. Create a global element named `SAP_Inbound` and specify the connection information.
+. Configure the required fields in the properties tab.
+. From the *Mule Palette* view, select *Core* and drag the *Transform Message* component to the right of *Function listener*.
+. Specify the details based on the metadata. For example:
++
+image::sap-function-return-response.png[Sample metadata for the SAP_Inbound global element]
+. Click *File > Save* to save the app.
+. Click *Run* > *Run as* > *Mule Application*.
++
```

Comment:
Please specify that this flow will display a result when a function will be trigger either from another mule flow or from sap gui.